src/main/java/seedu/flashcard/model/flashcard/Note.java

```diff
@@ -0,0 +1,39 @@
+package seedu.flashcard.model.flashcard;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents the Note on the Flashcard.
+ * Guarantees: immutable; is always valid.
+ */
+public class Note {
+    private String note;
+
+    /**
+     * Constructs a {@code Note}.
+     *
+     * @param note A valid note.
+     */
```

Comment:
Can add a regex checker to ensure the note is not an empty value. Reference Question / Answer class.

Reply:
The idea is that any note will be valid, just like the Remark class in tutorial, so that even if you want to remove a note from the Flashcard, you can do it using edit. Is it not supposed to be the case?